Sprint-6 Solution

llopFilms opened the pull request.

llopFilms committed:
Inicialització projecte amb Creat React App

llopFilms committed:
Exercici 1

llopFilms committed:
Exercici 2

llopFilms committed:
Exercici 3

llopFilms committed:
Exercici 4

llopFilms committed:
Exercici 5

llopFilms committed:
Exercici 6

llopFilms committed:
Millores finals + build + serve

llopFilms committed:
Sprint-6 Solution

monteserin commented on sprint6/src/App.js:
```diff
@@ -0,0 +1,38 @@
+import { FonsHistoriaGS } from "./components/globalStyles/FonsHistoriaGS";
+
+function App() {
+  const [inici, setInici] = useState(true);
```
Normalmente ponemos todos los hooks juntos y luego ya las funciones normales.

monteserin commented on sprint6/src/components/benvinguda/Benvinguda.jsx:
```diff
@@ -0,0 +1,28 @@
+
+export const Benvinguda = ({ eInici }) => {
+  return (
+    <ContainerS className="container-inici">
```
Si usas styled-componentes no usaría className

llopFilms committed:
Millores finals post-correcció:
- Carpeta inicial nova
- Doble README
- Carpetes de components amb maj
- Reset del build

llopFilms committed:
Millores finals 2:
- Arxius styled-component en maj + nom
- Hooks al principi de App
- Reset de build

llopFilms committed:
Github Pages deploy
